src/tests/test_updater.py

```diff
@@ -151,15 +146,14 @@ def test_taxonomy_common_name(self, multi_dbs):
 
     def test_fail_existing_genome_uuid_data_not_match(self, multi_dbs):
         test = meta_factory(multi_dbs['core_6'].dbc.url, multi_dbs['ensembl_genome_metadata'].dbc.url,
-                            multi_dbs['ncbi_taxonomy'].dbc.url)
+                            multi_dbs['ensembl_genome_metadata'].dbc.url)
```

Comment:
Does the call to `meta_factory` need two copies of `multi_dbs['ensembl_genome_metadata'].dbc.url`?

Reply:
it's not required , updated the test case